config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml

```diff
@@ -1,115 +1,135 @@
 presubmits:
   containerd/cri:
-  - name: pull-cri-containerd-build
+  - name: pull-cri-containerd-node-e2e
     always_run: true
+    max_concurrency: 8
+    decorate: true
     skip_branches:
     - v0.1
+    - release/1.0
+    - release/1.2
+    decoration_config:
+      timeout: 6000000000000 # 100m
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
     labels:
       preset-service-account: "true"
-    spec:
-      containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190125-3d9554697e-master
+      preset-k8s-ssh: "true"
```

Comment:
Does this work? I am not sure we verified this on the build controller

Reply:
:-) If you didn't make it to work, then it probably doesn't work